pipelines/matrix/src/matrix/pipelines/integration/pipeline.py

```diff
@@ -222,6 +222,15 @@ def create_pipeline(**kwargs) -> Pipeline:
                     name="check_merged_nodes_and_edges_consistency",
                     tags=["validation"],
                 ),
+                node(
+                    func=nodes.metric_abox_tbox,
```

Comment:
Not sure if this is the only issue though

```suggestion
                    func=nodes.compute_abox_tbox_metric,
```